dae/dae/utils/regions.py

```diff
@@ -424,7 +423,7 @@ def collapse_no_chrom(
 
 
 def total_length(regions: list[BedRegion]) -> int:
-    return sum(len(regions) for x in regions)
+    return sum(len(x) for x in regions)
```

Comment:
Does this function have a unit test?

Reply:
No. We need to add unit tests for most of the region utils.